client/app/app.less

```diff
@@ -6,4 +6,40 @@
       box-shadow: none;
     }
   }
+
+  .ip-clipboard {
+    min-width: rem-calc(200);
+    max-width: rem-calc(300);
+    margin: 0;
+  }
+
+  .font-inherit {
```

Comment:
Utilities should be in `ui-kit`

Reply:
This utility will not be useful with the next version of UI Kit. I let a comment about that.